.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: node_js
 node_js:
-- '10'
 - '12'
 install:
 - npm install
```

Comment:
If we're already changing this file, we should fix this to use CI rather than install (security purposes)

Reply:
So are we going to make all sites using the same travis from https://github.com/EclipseFdn/iot.eclipse.org/blob/master/.travis.yml ?

Reply:
Not necessarily? But using install over CI is a vulnerability as it can bring in unexpected versions of node packages.